prepline_general/api/general.py

```diff
@@ -746,7 +746,10 @@ def general_partition(
 
     # -- unzip any uploaded files that need it --
     for file_index in range(len(files)):
```

Comment:
Not sure what was the reason, it's old code and not mine. (few moments later) Ok I think it was because the file iterator can be replaced in last line. So iterating with index is needed, but also using enumarate syntax would work here. Normally I wouldn't refactor unrelated to PR code for beauty, but as I've already touched this loop, let's do it.

Reply:
Done.